src/main/java/org/jboss/qa/jenkins/test/executor/utils/MavenCli.java

```diff
@@ -417,10 +415,12 @@ public Builder params(Collection<String> params) {
 
 		public MavenCli build() {
 			if (mavenHome != null && !mavenHome.exists()) {
-				throw new IllegalArgumentException(String.format("Maven home does not exist: %s", mavenHome.getAbsolutePath()));
+				throw new IllegalArgumentException(String.format("Maven home does not exist: %s",
```

Comment:
@mswiech please do not split unchanged lines - there is no checkstyle restriction :)